frontend/src/loadPostHogJS.tsx

```diff
@@ -25,6 +26,11 @@ const configWithSentry = (config: Partial<PostHogConfig>): Partial<PostHogConfig
 
 export function loadPostHogJS(): void {
     if (window.JS_POSTHOG_API_KEY) {
+        const PREVIEW_FLAGS_V2_CONFIG = {
+            rolloutPercentage: 1,
+            includedHashes: new Set(['593cb24f9928bab39ec383c06c908481880d5099']),
```

Comment:
thinking about this more... I almost feel like I should pass this value in as a list of environment variables or something, feels like a better idea than committing hashed tokens (even if they are one-way secure).  

@haacked what do you think?  I could go either way – environment variables is more work bc more plumbing but arguably more secure.

Reply:
Yeah, it makes it easier if we have to back out, right?

Reply:
yup you were right the whole time; I was just tired when I implemented this.  After I got home and thought about it and read your comment I was like "huh I'm being an idiot"

Reply:
You're too hard on yourself. 😆